otter/test/integration/test_utils.py

```diff
@@ -0,0 +1,201 @@
+"""
+Tests for the utility functions for convergence black-box testing.
+"""
+from pyrsistent import pmap, pset
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from otter.convergence.model import (
```

Comment:
We're not doing any JSON marshalling.

Basically, we just needed some objects that describe all the state otter cares about, and re-using existing model classes is easier than writing new ones. These will never be the same instances as the functions in `otter.convergence` see, if that helps :)